bridge/framework/base.py

```diff
@@ -30,26 +30,26 @@ def is_remote(self) -> bool:
 
     def run(self) -> None:
         """Start services."""
-        if self.is_remote():
-            platform = detect_platform()
-        else:
-            platform = Platform.LOCAL
-            self.start_local_services()
+        platform = detect_platform() if self.is_remote() else Platform.LOCAL
         self.configure_services(platform)
```

Comment:
need to configure the services before they start for celery to detect the correct settings